### DISCUSSION TITLE
Enfoque a seguridad vial

### DISCUSSION THREAD

**Discussion by 2021030063** [Centinela-RR/Custodes#4] [Ideas]
A las palabras de @JW33D:
"[...] que tal si enfocamos la app 100% a seguridad vial para agregar una opción extra de S.O.S en caso de secuestros."
"Que usen el app a todo rato como el Google map y cuando estén en peligro solo den un click y uno de confirmación, osea a 2 click de seguridad"
Qué pensamos al respecto, @Centinela-RR/dev? En general, esto implica a todos pero fundamentalmente a los devs porque nos cargamos de trabajo jaj
Cómo se implementaría?
Qué implicaría?
Tendría que tener alteraciones a la db?
Cambiaría fundamentalmente el funcionamiento de la aplicación como tal?

**Comment by Yoel1515** [comment]
¿Sería una opción extra a lo que ya tenemos pensado?

**Reply by Yoel1515** [reply]
o literalmente es cambiar toda la aplicación?

**Reply by 2021030063** [reply]
Ese es el problema, apenas @JW33D para que aclare esto

**Reply by Yoel1515** [reply]
okeyy ahorita se manifiesta

**Reply by 2021030063** [reply]
Aunque como dijo "enfocar la app 100% en seguridad vial" lo veo como cambiar fundamentalmente la aplicación, o qué piensas?

**Reply by Yoel1515** [reply]
Una movida seria

**Comment by 2021030063** [comment]
Aunque la cosa es, no lo veo como algo posible o, idk, útil? Ya que al estar a punto de ser secuestrado (osea estar en peligro), no te pondrías a checar tu cel para alertar, y si tuvieses tiempo, usualmente estarías corriendo mientras llamas a 911

**Reply by Yoel1515** [reply]
seee es mas directo mejor llamar al 911

**Reply by JW33D** [reply]
En los carros ya tienen siempre prendido el Google maps y hasta compran aparatos para tenerlos a la vista, sería lo mismo

**Comment by JW33D** [comment]
Mi idea era agregar un botón en especifico tipo el Google Maps que solo dar 2 click lance un s.o.s

**Reply by Yoel1515** [reply]
Pero si pasa eso cualquier gente podría darle clic y como seria para saber que en realidad necesita ayuda

**Reply by JW33D** [reply]
Mi punto es ponerle un chorro de detalles para que le piensen 2 veces antes de estar jodiendo

**Reply by Yoel1515** [reply]
se escuha chido el chiste es saber como hacerlo jaja

**Reply by JW33D** [reply]
Eh ahí el detalle de idea,  y si quieren el S.O.S que no sea a la policía sino un aviso a contactos o perro que te ladre. Yo sé que es mucho pedir pero aún que sea que la  parte seguridad si quede 100% funcional

**Reply by Yoel1515** [reply]
Pues es buena idea de añadido que envie un mensaje de sos a un contacto que tengas, pero quien sabe los demás que piensen

**Comment by JW33D** [comment]
Sería más que nada dejar a un lado lo de servicios primarios y incendios para centrar en 3 módulos: bloqueos, estrellas y balaceras. Agregando el botón. Serían 4 cosas en total

**Comment by 2021030063** [comment]
Entonces, para concluir:

Cómo se implementaría?
- Agregando un botón junto al botón de reportar incidentes que sea de SOS

Qué implicaría?
- Eliminar el aspecto de reportes de servicios primarios e incendios para enfocarse únicamente en inseguridad

Tendría que tener alteraciones a la db?
- Sí

Cambiaría fundamentalmente el funcionamiento de la aplicación como tal?
- Sí, eliminaría un aspecto importante de ella

Algo más?

**Reply by Yoel1515** [reply]
Ya los problemas vendrán cuando se implemente o si falta algo

**Reply by 2021030063** [reply]
@JW33D Podrías dar la idea más concreta, para ponerla a votación?

**Reply by JW33D** [reply]
Mira yo creo que ya hice un desmadre jaja pero en mi mente así está la idea: una aplicación que la puedan tener abierta a todo rato al nivel del volante( tipo lo que hacen con Google maps) que tenga un botón de S.O.S para casos de secuestros que lance un mensaje con la ubicación si quieren a algún contacto guardado. Solo para que mínimo sepan dónde desapareció el wey o algo asi. Ya pensando más tranquilo se que es un pedo así que mínimo sea una aplicación para avisar donde hay un conflicto; porque creo que a la mayoría de conductores le vale madres si hay un poste chocado o una fuga y si existe Google ya avisa tráfico lento

**Reply by JW33D** [reply]
Si aún no está todo claro pos ya el lunes

**Reply by 2021030063** [reply]
Ya quedó, ya está la encuesta en #5 para el resto de la gente que ya esté en la org jaj

**Comment by 2021030063** [comment]
Aunque, ahora que lo pienso más, qué necesidad hay de quitar los demás módulos? Entiendo el querer hacerlo como un agregado pero, quitar la funcionalidad principal?
Tampoco entiendo claramente el por qué un conductor tendría la app abierta a todo momento; permitiría hacer rutas y la chingada como Google Maps? Porque si ese es el caso, literal estaríamos copiando Google Maps, y si no es el caso, por qué la tendrían abierta?

**Reply by JW33D** [reply]
Buen punto, pos que se quede como está jaja

**Comment by Yoel1515** [comment]
Se me hace más innovador lo que ya teniamos

**Reply by 2021030063** [reply]
Claro, podemos agregar la opción ya que de hecho está muy alineada a lo que teníamos en mente inicialmente, pero no veo para qué eliminar funcionalidades. Quizás para dedicarlo específicamente a un solo tema? 🤔 
Tendré que cambiar la encuesta 💀

**Reply by JW33D** [reply]
Nombre wey así déjale, solo era sugerencia así a las prisas. Estaba con un ojo en el taller y el otro tratando de contestarle a ustedes jaja

**Reply by 2021030063** [reply]
Mejor agreguemos la opción y agreguemos una opción para desactivar vistas que no interesen 
Qué te parece? Eh? Eh? Sí o no soy un genio para que quede mejor la app sin comprometer las contribuciones de nadie? Eh? EH?????;DDDD
Poll updated btw, vote again pls

**Reply by JW33D** [reply]
Ufff por eso eres el líder wey
